qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -0,0 +1,14 @@
+- x86_cpu_flags:
+    type = x86_cpu_flags
+    cpu_model_flags += ",enforce"
```

Comment:
Maybe you could add '-mpx' here, so you don't have to set it for every case in the future.

Reply:
update and thanks.